### PR TITLE
pallet-referenda: make the pallet instantiable

### PR DESCRIPTION
**Pull request by koushiro**
Signed-off-by: koushiro <koushiro.cqx@gmail.com>

related issue: #11056

polkadot address: 15XuanNimo5951s3RjFTPX1AvYVnCwfr3SDhb3AV4fQF3LpK

### CODE REVIEW

File: frame/referenda/src/lib.rs [hunk 1]
```diff
@@ -83,15 +83,18 @@ use sp_std::{fmt::Debug, prelude::*};
...
-use branch::{BeginDecidingBranch, OneFewerDecidingBranch, ServiceBranch};
-pub use pallet::*;
-pub use types::{
-	BalanceOf, CallOf, Curve, DecidingStatus, DecidingStatusOf, Deposit, InsertSorted,
-	NegativeImbalanceOf, PalletsOriginOf, ReferendumIndex, ReferendumInfo, ReferendumInfoOf,
-	ReferendumStatus, ReferendumStatusOf, ScheduleAddressOf, TallyOf, TrackIdOf, TrackInfo,
-	TrackInfoOf, TracksInfo, VotesOf,
+
+use self::branch::{BeginDecidingBranch, OneFewerDecidingBranch, ServiceBranch};
+pub use self::{
```
Comment (shawntabrizi) [inline frame/referenda/src/lib.rs:88]: why self?

**Review by shawntabrizi — APPROVED** [review]

**Comment by shawntabrizi** [comment]
delivered a tip in the conviction-voting pr

File: frame/referenda/src/lib.rs (hunk 1 above)
Comment (koushiro) [reply frame/referenda/src/lib.rs:88]: Adding `self` or `crate` makes it easier to distinguish `module` name from `external crate` name, I think

**Review by ggwpez — APPROVED** [review]

**Comment by shawntabrizi** [comment]
bot merge

**Pull request merged**